serving/docker/Dockerfile

```diff
@@ -23,6 +23,12 @@ EXPOSE 8080
 
 WORKDIR /opt/djl
 ENV JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
+ENV OMP_NUM_THREADS=1
+ENV TF_NUM_INTEROP_THREADS=1
+ENV TF_NUM_INTRAOP_THREADS=1
+ENV TF_CPP_MIN_LOG_LEVEL=1
+ENV MXNET_ENGINE_TYPE=NaiveEngine
+ENV JAVA_OPTS="-Dai.djl.disable_close_resource_on_finalize=true"
```

Comment:
-Dai.djl.pytorch.num_interop_threads=1 ?

Reply:
Do we need ai.djl.pytorch.num_interop_threads? I thought it has no impact

Reply:
From what we observed on resnet18 yes. But it should have impact when operators in the model can be run concurrently

Reply:
We can change it later if we find one